Eatery Blue.xcodeproj/project.pbxproj

```diff
@@ -7,7 +7,6 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		2E08E1952AD469BB0036E0E1 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 2E08E1942AD469BB0036E0E1 /* GoogleService-Info.plist */; };
```

Comment:
I will work with you and @antoinette-torres (maybe during work session) to prevent this from happening. You both will need to make sure that your branches don't have this issue.

Reply:
yes usually i only stage the files that i changed code in but this time i kind of got lazy and did "add .". I'll be sure to get in the habit of that from now on.